bindings/core/src/method/utils.rs

```diff
@@ -159,4 +162,10 @@ pub enum UtilsMethod {
     /// Returns the hex representation of the serialized output bytes.
     #[serde(rename_all = "camelCase")]
     OutputHexBytes { output: OutputDto },
+    /// Verifies the semantic of a transaction.
+    VerifySemantic {
```

Comment:
Maybe we should call it VerifyTransactionSemantic 🤔

Reply:
Fine for me, but we can only rename in the bindings